<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Precio de libros</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="precioLibros"/>
      </item>
      <item>
       <widget class="QSpinBox" name="cantidadLibros">
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="totalLibros"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Precio del pan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="precioPan"/>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="cantidadPan">
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="totalPan"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="total">
      <property name="text">
       <string>Total:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="calcular">
      <property name="text">
       <string>Calcular</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
